src/main/scala/package.scala

```diff
@@ -4,13 +4,26 @@ import ohnosequences.awstools.s3._
 
 package object rna16s {
 
-  val dbName = "ohnosequences.db.rna16s"
+  type +[A, B] = Either[A, B]
 
-  private val metadata = generated.metadata.rna16s
+  implicit class PredicateOps[X](val p: X => Boolean) {
+
+    def &&(other: X => Boolean): X => Boolean =
+      x => p(x) && other(x)
+  }
+
+  val dbName: String =
+    "ohnosequences.db.rna16s"
+
+  val asPath: String => String =
+    _.replace('.', '/')
+
+  // val metadata: ohnosequences.statika.AnyArtifactMetadata =
+  //   ohnosequences.db.generated.metadata.rna16s
 
   val s3prefix: S3Folder =
     s3"resources.ohnosequences.com" /
-    metadata.organization /
-    metadata.artifact /
-    metadata.version /
+      asPath(dbName) /
+      "argh" /
```

Comment:
Why `argh`? I mean, why a subdirectory inside `rna16s`? And we are changing this name, right? :laughing:

Reply:
it's a reminder of sorts; I need to sort this out.

Reply:
done.